packages/create-cloudflare/src/frameworks/index.ts

```diff
@@ -49,4 +50,22 @@ export const runFrameworkGenerator = async (ctx: C3Context, args: string[]) => {
 	logRaw("");
 
 	await runCommand(cmd, { env });
+
+	// When running e2e tests, commit the result of the scaffolding tool to facilitate
+	// diffing what new code is added by C3 as part of the process
+	if (process.env.SAVE_DIFFS) {
+		const cmdEnv = {
+			silent: true,
+			cwd: ctx.project.path,
+		};
+
+		// Certain framework scaffolders commit by default. If that's the case, skip.
```

Comment:
Leftover from a previous implementation, good catch. I'll remove

Reply:
It is. For example, `create-react-app` commits by default. In that case, we don't want to try the following code, because the git commit will fail. Thus, we're checking if the `create-*` package committed already by using `isInsideGitRepo` before proceeding.

Reply:
ah ok, because a `create-*` tool will only create a git repo when it actually commits?

if that's the case maybe we could update the comment to reflect that?

as in:
```suggestion
		// Certain framework scaffolders commit by default, this implies that they initialize a git repo.
		// so If that's the case and we are in a repo we skip.
```